tests/CMakeLists.txt

```diff
@@ -32,7 +32,11 @@ set_target_properties(check-ia2 PROPERTIES FOLDER "tests")
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
-# Set up 'check' target and check-exec on arm
+# Set up 'check' target
+if (LIBIA2_AARCH64)
+    # On aarch64, avoid swallowing QEMU MTE check failures
+    set(CMAKE_CTEST_COMMAND ${CMAKE_CTEST_COMMAND} --verbose)
```

Comment:
Why not do this on x86 too?

Reply:
> Our QEMU fork effectively runs in permissive mode, so this is necessary to distinguish test runs with violations from those without.

Right, we need this for tests that fail due to unintentional violations. I don't remember exactly what currently happens with `CHECK_VIOLATION` on ARM off the top of my head.